lib/web/mage/menu.js

```diff
@@ -440,6 +440,12 @@ define([
                     if (!target.hasClass('level-top') || !target.has('.ui-menu').length) {
                         window.location.href = target.find('> a').attr('href');
                     }
+
+                    submenu = target.find('ul.submenu').first();
+
+                    if (submenu.attr('aria-expanded') ==) 'true') {
```

Comment:
It seems like there is a typo now. Please double check, I think you meant:

```suggestion
                    if (submenu.attr('aria-expanded') === 'true') {
```